composables/zksync/useWithdrawalFinalization.ts

```diff
@@ -70,16 +71,16 @@ export default (transactionInfo: ComputedRef<TransactionInfo>) => {
     finalizeWithdrawalParams.value = await getFinalizationParams();
     if (usingMainContract.value) {
```

Comment:
Looks like we can remove this if statement since both ETH and ERC20 use same method and contract, right?

Reply:
Yes, you are right!